src/libraries/System.Formats.Tar/tests/TarEntry/PaxTarEntry.Conversion.Tests.cs

```diff
@@ -92,8 +92,8 @@ public void Constructor_ConversionFromV7_Write(TarEntryFormat originalEntryForma
             dataStream.Position = 0;
             originalEntry.DataStream = dataStream;
 
-            DateTimeOffset expectedATime = default;
-            DateTimeOffset expectedCTime = default;
+            DateTimeOffset? expectedATime = null;
+            DateTimeOffset? expectedCTime = null;
 
             if (originalEntry is GnuTarEntry gnuEntry)
```

Comment:
Should we include a case for a Gnu entry where these aren't set, to ensure we don't set them in the Pax entry when converting?

Reply:
I'll be adding some more cases to the conversion tests. Also for the round-tripping.

Reply:
This is done. The existing conversion tests have atime/ctime not set, and two new tests check the behavior when atime/ctime are set on Gnu and Pax.